applications/potree-viewer/tapisjob_app.sh

```diff
@@ -0,0 +1,57 @@
+handle_error() {
+  local EXITCODE=$1
+  echo "Potree Viewer job exited with an error status. $EXITCODE" >&2
+  echo "$EXITCODE" > "${_tapisExecSystemOutputDir}/tapisjob.exitcode"
+  exit $EXITCODE
+}
+
+set -x
+echo "TACC: job $_tapisJobUUID execution at: `date`"
+
+LOGIN_PORT=$(( ((RANDOM<<15)|RANDOM) % 100 + 5900 ))
+quit=0
+
+while [ "$quit" -ne 1 ]; do
+  netstat -a | grep $LOGIN_PORT >> /dev/null
+  if [ $? -gt 0 ]; then
+    quit=1
+  else
+    LOGIN_PORT=`expr $LOGIN_PORT + 1`
+  fi
+done
+echo "Using port=$LOGIN_PORT"
+
+POTREE_INSTANCE_NAME="potree-viewer-${_tapisJobUUID}"
+
+apptainer instance run \
+    --writable-tmpfs \
+    --memory 1G \
+    --bind ${_tapisExecSystemInputDir}:/data \
+    --bind /opt/potree:/potree \
+    --bind /etc/letsencrypt/live/wma-exec-01.tacc.utexas.edu/cert.pem:/etc/nginx/ssl/nginx.crt \
+    --bind /etc/letsencrypt/live/wma-exec-01.tacc.utexas.edu/privkey.pem:/etc/nginx/ssl/nginx.key \
+    --env LOGIN_PORT=$LOGIN_PORT \
+    docker://taccaci/potree-viewer:1.8.2 $POTREE_INSTANCE_NAME \
+    /bin/bash -c "cp -r /potree/{build,libs} /data/ && envsubst < /etc/nginx/conf.d/default.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+
+# Webhook callback url for job ready notification
+# (notifications sent to INTERACTIVE_WEBHOOK_URL (i.e. https://3dem.org/webhooks/interactive/))`
+INTERACTIVE_WEBHOOK_URL="${_INTERACTIVE_WEBHOOK_URL}"
```

Comment:
Here a new variable is declared, but on line 42, it uses `_INTERACTIVE_WEBHOOK_URL`, i.e. not this new declared variable. You can forego declaring this variable